src/org/parosproxy/paros/Constant.java

```diff
@@ -81,6 +81,7 @@
 // ZAP: 2018/06/29 Allow to check if in dev mode.
 // ZAP: 2018/07/19 Fallback to bundled config.xml and log4j.properties.
 // ZAP: 2019/03/14 Move and correct update of old options.
+// ZAP: 2019/04/01 Refactored getContextsDir, getPoliciesDir into getDir.

```

Comment:
This no longer contains the relevant details.

Reply:
@kingthorin fixed, thank you.